Source/WebCore/page/Navigation.cpp

```diff
@@ -546,13 +546,16 @@ void Navigation::cleanupAPIMethodTracker(NavigationAPIMethodTracker* apiMethodTr
 // https://html.spec.whatwg.org/multipage/nav-history-apis.html#abort-the-ongoing-navigation
 void Navigation::abortOngoingNavigation(NavigateEvent& event)
 {
+    if (!scriptExecutionContext()->globalObject() && !m_ongoingAPIMethodTracker)
+        return;
+
```

Comment:
This seems fine. Just for clarity I'd probably write it a bit different, like:

```cpp
auto* globalObject = ...;
if (!globalObject)
    return;

...
```

Instead of the ambiguous check of `!m_ongoingAPIMethodTracker`.